Dockerfile

```diff
@@ -60,7 +62,10 @@ RUN /usr/src/aws/install --bin-dir /usr/bin
 ADD https://raw.githubusercontent.com/CosmWasm/wasmvm/$WASMVM_VERSION/api/libwasmvm.so /lib/libwasmvm.so
 
 COPY run.sh /usr/bin/
+COPY backup_manager.sh /usr/bin/
 RUN chmod +x /usr/bin/run.sh
+RUN chmod +x /usr/bin/backup_manager.sh
 ENTRYPOINT ["run.sh"]
 
-CMD $PROJECT_BIN start
+# CMD $PROJECT_CMD
+CMD backup_manager.sh
```

Comment:
The snapshot script should be used as a `command` override but might need to work around that until Akash supports the `command` argument correctly https://github.com/ovrclk/akash/issues/1301

Reply:
I see you have finished.

Have you made this optional? Or is it baked in to a degree it cannot be stopped?

Reply:
Still WIP and it'll be entirely optional. The script will be bundled in to the container image but won't be used unless the right ENV var is set.